todo_app/tests/test_integration.py

```diff
@@ -1,46 +1,25 @@
 from todo_app import app
-from unittest.mock import patch
-from unittest.mock import Mock
 
 import pytest
 import dotenv
-import os
-
-sample_trello_cards_response = [{'id': '60c71900c47a8259cb2c912d', 'idList': '609542268e084d62bd913af8','name': 'Test Card'}]
-sample_trello_lists_response = [{'id': '609542268e084d62bd913af8', 'name': 'Doing'}]
+import mongomock
 
 @pytest.fixture
 def client():
     # Use our test integration config instead of the 'real' version
     file_path = dotenv.find_dotenv('.env.test')
     dotenv.load_dotenv(file_path, override=True)
 
-    # Create the new app.
-    test_app = app.create_app()
-
     # Use the app to create a test_client that can be used in our tests.
-    with test_app.test_client() as client:
-        yield client
+    with mongomock.patch(servers=(('fakemongo.com', 27017),)):
+        test_app = app.create_app()
+        with test_app.test_client() as client:
+            yield client 
+
 
-@patch('requests.get')
-def test_index_page(mock_get_requests, client):
-    # Replace call to requests.get(url) with our own function
-    mock_get_requests.side_effect = mock_get
+def test_index_page(client):
+    client.post('/items', data=dict(title='Test Card', status='To Do'))
```

Comment:
Nice, we could use your existing code to add a new item directly to the fake DB, but having it instead POST allows us to test both endpoints